ci-operator/step-registry/openshift/e2e/test/pausedworkers/openshift-e2e-test-pausedworkers-commands.sh

```diff
@@ -0,0 +1,210 @@
+#!/bin/bash
```

Comment:
Completely cloning e2e-test `commands.sh` is a bit of code smell, is this absolutely necessary?

Reply:
Roughly, what's your plan for not having to do this for very long (keep a completely divergent copy)?

Reply:
Ideally CI would allow importing some scripts between steps, this would be beneficial for other steps.

A short-term approach would be introducing a new `TEST_TYPE=update-paused` (let me rework this PR to use it)